00:00:00.001227 main     VirtualBox COM Server 6.1.16 r140961 win.amd64 (Oct 16 2020 16:33:17) release log
00:00:00.001230 main     Log opened 2020-11-23T15:22:43.586742300Z
00:00:00.001231 main     Build Type: release
00:00:00.001234 main     OS Product: Windows 10
00:00:00.001235 main     OS Release: 10.0.18363
00:00:00.001235 main     OS Service Pack: 
00:00:00.080832 main     DMI Product Name: 20L6S9U200
00:00:00.088208 main     DMI Product Version: ThinkPad T480
00:00:00.088225 main     Firmware type: BIOS
00:00:00.088233 main     Host RAM: 32426MB (31.6GB) total, 23655MB (23.1GB) available
00:00:00.088236 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.088237 main     Process ID: 13984
00:00:00.088237 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.168391          VirtualBox: object creation starts
00:00:00.169498          Home directory: 'F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox'
00:00:00.169606          Installed Drivers:
00:00:00.173726            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.16.40961)
00:00:00.174068            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.16.40961)
00:00:00.174298            C:\WINDOWS\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.16.40961)
00:00:00.174707            C:\WINDOWS\System32\drivers\VBoxNetAdp6.sys (Version: 6.1.16.40961)
00:00:00.177575          Loading settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.178954          usbLibGetDevices: Starting USB device enumeration
00:00:00.180360          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.180402          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 7
00:00:00.180438          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:00:00.180473          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 9
00:00:00.181510          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 15
00:00:00.181669          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 2
00:00:00.181703          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.182984          usbLibGetDevices: Found 10 USB devices, 0 captured
00:00:00.183068          USB: Unknown USB device detected (idVendor: 0x06cb, idProduct: 0x009a)
00:00:00.183081          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x0316)
00:00:00.183096          USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x56a6)
00:00:00.183113          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0a2b)
00:00:00.183122          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0ac9)
00:00:00.183137          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0aca)
00:00:00.260735          HostDnsMonitor: initializing
00:00:00.269322          HostDnsMonitor: updating information
00:00:00.269348          HostDnsMonitor: old information
00:00:00.269349            no server entries
00:00:00.269350            no domain set
00:00:00.269350            no search string entries
00:00:00.269350          HostDnsMonitor: new information
00:00:00.269351            server 1: 127.0.0.1
00:00:00.269352            domain: ni.corp.natinst.com
00:00:00.269352            no search string entries
00:00:00.366789          VD: VDInit finished with VINF_SUCCESS
00:00:00.367739          Loading settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:00:00.368910          Loading settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\Debug\Debug.vbox" with version "1.14-windows"
00:00:00.378685          OCI: Local config file 'F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\oci_config' does not exist
00:00:00.378791          OCI: Original config file 'C:\Users\inthirua\.oci\config' does not exist
00:00:00.378792          OCI: Reading profiles finished with status ERROR_SUCCESS
00:00:00.378799          ExtPack: Created cloud provider 'OCI' (hrc=ERROR_SUCCESS)
00:00:00.378821          VirtualBox: object created
00:00:08.756679          Launched VM: 63816560 pid: 3388 (0xd3c) frontend: GUI/Qt name: LinuxLive
00:00:12.382133          Load [C:\Program Files\Oracle\VirtualBox\ExtensionPacks\Oracle_VM_VirtualBox_Extension_Pack\win.amd64\VBoxHostWebcam.DLL] vrc=VINF_SUCCESS
00:00:12.613637          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:07:00.116945          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:07:06.002246          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:07:06.088429          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:07:06.174355          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:07:06.253834          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:07:06.337320          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:07:06.417456          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:07:06.501427          Saving settings file "F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:07:12.090493 main     VirtualBox: object deletion starts
00:07:12.433386 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'F:\PortableApps\VirtualBox\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\HardDisks/LinuxLive.vmdk' cannot be closed because it is still attached to 2 virtual machines}, preserve=false aResultDetail=0
00:07:12.438982 main     HostDnsMonitor: shutting down ...
00:07:12.443909 main     HostDnsMonitor: shut down
00:07:12.451952 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
00:07:12.458060 main     VirtualBox: object deleted
